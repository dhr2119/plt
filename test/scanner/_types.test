int double bool void null String true false
