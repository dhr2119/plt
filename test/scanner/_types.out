INT
DOUBLE
BOOL
VOID
NULL
STRING
TRUE
FALSE
